platform/string_storage_base.hpp

```diff
@@ -12,13 +12,13 @@ class StringStorageBase
   explicit StringStorageBase(std::string const & path);
   void Save() const;
   void Clear();
-  bool GetValue(std::string const & key, std::string & outValue) const;
-  void SetValue(std::string const & key, std::string && value);
+  bool GetValue(std::string_view key, std::string & outValue) const;
+  void SetValue(std::string_view key, std::string && value);
   void Update(std::map<std::string, std::string> const & values);
-  void DeleteKeyAndValue(std::string const & key);
-  
+  void DeleteKeyAndValue(std::string_view key);
+
 private:
-  using Container = std::map<std::string, std::string>;
+  using Container = std::map<std::string, std::string, std::less<>>;
```

Comment:
Why explicit less? Isn't it already defined?

Reply:
is_transparent find